python/bacalhau_sdk/job_store.py

```diff
@@ -0,0 +1,152 @@
+from bacalhau_sdk.orchestrator_service import OrchestratorService
+from bacalhau_apiclient.models.api_put_job_response import (
+    ApiPutJobResponse as PutJobResponse,
+)
+from bacalhau_apiclient.models.api_put_job_request import (
+    ApiPutJobRequest as PutJobRequest,
+)
+from bacalhau_apiclient.models.api_stop_job_response import (
+    ApiStopJobResponse as StopJobResponse,
+)
+from bacalhau_apiclient.models.api_list_job_executions_response import (
+    ApiListJobExecutionsResponse as ListJobExecutionsResponse,
+)
+from bacalhau_apiclient.models.api_list_job_results_response import (
+    ApiListJobResultsResponse as ListJobResultsResponse,
+)
+
+from bacalhau_apiclient.models.api_list_job_history_response import (
+    ApiListJobHistoryResponse as ListJobHistoryResponse,
+)
+from bacalhau_apiclient.models.api_list_jobs_response import (
+    ApiListJobsResponse as ListJobsResponse,
+)
+from bacalhau_apiclient.models.api_get_job_response import (
+    ApiGetJobResponse as GetJobResponse,
+)
+
+from bacalhau_sdk.config import init_config
+
+
+class JobStore:
```

Comment:
Nice structure of resource based SDKs. Though can we call them `Jobs` similar to our [go sdk](https://github.com/bacalhau-project/bacalhau/blob/main/pkg/publicapi/client/v2/api_jobs.go#L13). `JobStore` is in internal component related to persisting and retrieving job state, and can be confusing here